ckanext/csrf_filter/anti_csrf.py

```diff
@@ -90,6 +91,9 @@ def configure(config):
     token_expiry_age = 60 * config.get('ckanext.csrf_filter.token_expiry_minutes', 30)
     token_renewal_age = 60 * config.get('ckanext.csrf_filter.token_renewal_minutes', 10)
 
+    exempt_rules = config.get('ckanext.csrf_filter.exempt_rules', None)
+    if exempt_rules:
+        exempt_rules = [re.compile(rule) for rule in exempt_rules.split()]
```

Comment:
So, splitting by whitespace?

Reply:
Changed it to be a JSON list. Seems better and is easier to read an write in INI files. Added this info to the readme for the config option